tests/TestCase/Database/Type/DateTypeTest.php

```diff
@@ -75,11 +75,11 @@ public function testToDatabase()
         $result = $this->type->toDatabase($value, $this->driver);
         $this->assertEquals($value, $result);
 
-        $date = new Time('2013-08-12');
+        $date = new Date('2013-08-12');
         $result = $this->type->toDatabase($date, $this->driver);
         $this->assertEquals('2013-08-12', $result);
 
-        $date = new Time('2013-08-12 15:16:18');
+        $date = new Date('2013-08-12 15:16:18');
         $result = $this->type->toDatabase($date, $this->driver);
```

Comment:
We should probably keep a test usimg Time to make sure we don't make mistakes in the future.